File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/PHY2001U04RefreshCounterInfo.java
package nta.med.data.model.ihis.phys;

import java.math.BigInteger;

public class PHY2001U04RefreshCounterInfo {
	private BigInteger g1 ;
	private BigInteger g2;
	private BigInteger sa;
	private BigInteger re;
	public PHY2001U04RefreshCounterInfo(BigInteger g1, BigInteger g2,
			BigInteger sa, BigInteger re) {
		super();
		this.g1 = g1;
		this.g2 = g2;
		this.sa = sa;
		this.re = re;
	}
	public BigInteger getG1() {
		return g1;
	}
	public void setG1(BigInteger g1) {
		this.g1 = g1;
	}
	public BigInteger getG2() {
		return g2;
	}
	public void setG2(BigInteger g2) {
		this.g2 = g2;
	}
	public BigInteger getSa() {
		return sa;
	}
	public void setSa(BigInteger sa) {
		this.sa = sa;
	}
	public BigInteger getRe() {
		return re;
	}
	public void setRe(BigInteger re) {
		this.re = re;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY2001U04RefreshCounterInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
    public class PHY2001U04RefreshCounterInfo
    {
        private String _g1;
        private String _g2;
        private String _sa;
        private String _re;

        public String G1
        {
            get { return this._g1; }
            set { this._g1 = value; }
        }

        public String G2
        {
            get { return this._g2; }
            set { this._g2 = value; }
        }

        public String Sa
        {
            get { return this._sa; }
            set { this._sa = value; }
        }

        public String Re
        {
            get { return this._re; }
            set { this._re = value; }
        }

        public PHY2001U04RefreshCounterInfo() { }

        public PHY2001U04RefreshCounterInfo(String g1, String g2, String sa, String re)
        {
            this._g1 = g1;
            this._g2 = g2;
            this._sa = sa;
            this._re = re;
        }

    }
}